migrations/20171104220247_add_heilas_table.js

```diff
@@ -9,7 +9,8 @@ exports.up = function(knex, Promise) {
         .onDelete('RESTRICT')
         .onUpdate('CASCADE');
 
-      table.string('image_path');
+      table.string('bio_text', 250);
+      table.string('bio_looking_for', 250);
```

Comment:
These changes should be in a new migration file.

Reply:
Yes, that's true. I decided to nuke my db since nothing's migrated to the real master @ Futu Heroku. Since this was something completely reverted I thought it would work just fine this way.